build.xml

```diff
@@ -246,19 +246,36 @@
         
         <copy file="${basedir}/LICENSE.txt" todir="${basedir}/vscode" />
         <copy file="${basedir}/THIRD_PARTY_LICENSES.txt" todir="${basedir}/vscode" />
-        
+        <copy file="${basedir}/README.md" todir="${basedir}/vscode" />
```

Comment:
@shivam71 i suppose image resource paths also needs to be corrected accordingly when we move readme.md from root dir to vscode dir. could you check once?
`![Enable Preview](vscode/images/enable_preview.gif) has to  get converted to ![Enable Preview](images/enable_preview.gif) `

Reply:
or bit cleaner approach is  we can maintain images in root dir level as a seperate dir and copy images dir also to vscode dir - this might need bit of refactoring.

Reply:
Thanks for noticing that ! , since while packaging the relative urls are converted to https GitHub urls this isn't an issue .